components/remote_settings/src/service.rs

```diff
@@ -81,31 +79,6 @@ impl RemoteSettingsService {
         Ok(client)
     }
 
-    #[cfg(not(feature = "jexl"))]
```

Comment:
I'm not sure `jexl` has to be an optional feature. But leaving it to your appreciation

Reply:
I think we should probably enable jexl for Android/iOS at least, as they may need it for filtering some collections - even if we don't use it filtering up-front, we might need it later, so this would avoid some work when adding it & having to remember that older clients wouldn't necessarily have support for it.

For desktop, I'm not sure if we're using this or not yet, but it would seem a good idea to get it enabled.

Do I remember correctly that we needed to do a bit of extra work to enable it on desktop - some sort of crate review or something?

Reply:
I think it's probably just auditing the crates.  I'll check on this and see how much work it is.